### ISSUE TITLE
Separate GUI (Swing) classes from Main class

### ISSUE THREAD

**Issue by edigonzales** [claeis/ili2db#357]
Damit die Anwendung mit GraalVM zu native images kompilierbar ist, dürfen keine Swing-Abhängigkeiten in der Main-Klasse vorhanden sein.

**Comment by claeis** [comment]
NO-OP Versionen der GUI Klassen sind in gui-noop/src (statt gui/src)

**Comment by edigonzales** [comment]
Soll man das noch steuern können via Property?

```
gradle clean compileIli2pgJava -Pnogui
```

```
def nogui = hasProperty('nogui') ? true : false

sourceSets {
.....
    java {
        if (nogui) {
    	    srcDirs=['src','gui-noop/src'] 
        } else {
            srcDirs=['src','gui/src']
        }
    }
....
}
```

**Comment by claeis** [comment]
Das möchte ich nicht machen. So bedingte Build-Skripts erzeugen zuviel Komplexität.
Ich stelle mir im Moment eher eine Lösung vor, wo ili2db:AbstractMain schaut, ob die GUI Klasse vorhanden ist (via ```Class.forName(guiClassName).newInstance();```).
Ich schaus wieder an, wenn ich claeis/ilivalidator#222 implementiere.

**Comment by edigonzales** [comment]
Wie gehen wir mit den benötigten Konfig-Dateien um? Beispiel: https://github.com/edigonzales/ili2c-native/tree/master/src/main/resources/META-INF/native-image 

Wer ist für diese verantwortlich? Das Projekt "claeis/ili2db" oder die Organisation/Person, welche native images macht?
